hack/patches/kustomize-controller-deploy-patch.yaml

```diff
@@ -16,6 +16,6 @@
 - op: replace
   path: /spec/template/metadata/annotations
   value: '{{ include "podTemplateAnnotations.kustomizeController" . }}'
-- op: replace
-  path: /spec/template/metadata/labels
-  value: '{{ include "podTemplateLabels.kustomizeController" . }}'
+#- op: replace
```

Comment:
This had to be removed, cos it does work. The `kubectl kustomize` command fails that you try to replace a map with a scalar value.